### PR TITLE
Update applications.yaml

### PR DESCRIPTION
**Pull request by jonathan-shopware**
Improve fastly installation to not run in Dev projects when they have environment_type production

### CODE REVIEW

File: shopware/paas-meta/6.6/.platform/applications.yaml [hunk 1]
```diff
@@ -174,7 +174,8 @@
             echo "==================================================="
             echo "SETUP FASTLY AND CLEAR OPCACHE"
             echo "==================================================="
-            if [ "$PLATFORM_ENVIRONMENT_TYPE" = "production" ] || [ "$PLATFORM_ENVIRONMENT_TYPE" = "stage" ]; then
+            # If Fastly should be enabled on stage or live instance it is required to add it as env variable (works with "live" or "stage" as value)
+            if [ "$FASTLY_ENV_TYPE" = "live" ] || [ "$FASTLY_ENV_TYPE" = "stage" ]; then
```
Comment (shyim) [inline shopware/paas-meta/6.6/.platform/applications.yaml:178]: is that something what psh sets, or you have to set it?

**Comment by jonathan-shopware** [comment]
PLATFORM_ENVIRONMENT_TYPE is automatically there, and often, branches are marked as environment_type production even if those are dev environments. The Variable FASTLY_ENV_TYPE would be a custom variable that needs to be added either in applications.yaml or via environment variable directly in the console interface. But mostly this step is only required then on the dedicated instances which have a real domain name as well.

File: shopware/paas-meta/6.6/.platform/applications.yaml (hunk 1 above)
Comment (jonathan-shopware) [reply shopware/paas-meta/6.6/.platform/applications.yaml:178]: PLATFORM_ENVIRONMENT_TYPE is automatically there, and often, branches are marked as environment_type production even if those are dev environments. The Variable FASTLY_ENV_TYPE would be a custom variable that needs to be added either in applications.yaml or via environment variable directly in the console interface. But mostly this step is only required then on the dedicated instances which have a real domain name as well.

Comment (jonathan-shopware) [reply shopware/paas-meta/6.6/.platform/applications.yaml:178]: So can be set by psh and by the developer which works on the project himself.

Comment (shyim) [reply shopware/paas-meta/6.6/.platform/applications.yaml:178]: we need documentation. i would not like to rename it and then the user does not understand why it didn't get executed

**Pull request merged**